mods/sgssb/moves.js

```diff
@@ -418,12 +418,24 @@ exports.BattleMovedex = {
 		isNonstandard: true,
 		name: "Mewtation",
 		pp: 10,
+		secondary: {
+			chance: 100,
```

Comment:
If you do `chance: 100,` Might as well take the thing out of secondary and put in `status: tox ',` and put `secondary: false,`